soh/src/code/z_parameter.c

```diff
@@ -3125,6 +3125,15 @@ void Interface_Draw(GlobalContext* globalCtx) {
     };
     static s16 rupeeDigitsFirst[] = { 1, 0, 0 };
     static s16 rupeeDigitsCount[] = { 2, 3, 3 };
+
+    // courtesy of https://github.com/TestRunnerSRL/OoT-Randomizer/blob/Dev/ASM/c/hud_colors.c
+    static s16 rupeeWalletColors[4][3] = {
+        { 0xC8, 0xFF, 0x64 }, // Base Wallet (Green)
+        { 0x82, 0x82, 0xFF }, // Adult's Wallet (Blue)
+        { 0xFF, 0x64, 0x64 }, // Giant's Wallet (Red)
+        { 0xFF, 0x5A, 0xFF }, // Tycoon's Wallet (Purple)
+    };
```

Comment:
```suggestion
    static s16 rupeeWalletColors[4][3] = {
        { 0xC8, 0xFF, 0x64 }, // Base Wallet (Green)
        { 0x82, 0x82, 0xFF }, // Adult's Wallet (Blue)
        { 0xFF, 0x64, 0x64 }, // Giant's Wallet (Red)
    };
```
We dont have that wallet so we dont need this one.

Reply:
Shouldn't the dimensions be [3][3] as well?

Reply:
yes they should be updated.